code/modules/antagonists/traitor/backstory/traitor_backstories.dm

```diff
@@ -0,0 +1,124 @@
+/datum/traitor_backstory
+	/// The name of this traitor backstory type, displayed as a title
+	var/name
+	/// A description of the events leading up to this traitor's existence
+	var/description
+	/// Factions you can have as this backstory
+	var/allowed_factions = list(
+		TRAITOR_FACTION_BLACK_MARKET,
+		TRAITOR_FACTION_SYNDICATE
+	)
+	/// A list of motivation types for this backstory, used for filtering and searching
+	var/list/motivations = list()
+	/// If this backstory suggested for murderboning or hijacking
+	var/murderbone = FALSE
```

Comment:
Personally I think all of these objectives work fine with murderbone objectives, if you are doing it because of a debtor it still makes sense that the syndicate may give you the objective to hijack or something. As long as you are working for the syndicate, you have to do their missions and follow their demands

Reply:
Oh wait, are the objectives decided prior to backstory selection and this is just a recommendation flag? I was mistaken and thought that the objectives were decided based on what you pick

Reply:
It's all recommendations. The flag only recommends against using 'forced' backstories for murderbone, because the syndicate knows you're probably a wuss and can't handle it. However the player can still select those backstories.

It meshes better that way imo, but again, not a requirement. The tool is designed to help you pick the best backstory for your character and objectives, but you can always pick whatever.

Reply:
Ah okay, if they are only used to guide recommendations then it makes more sense. The murderbone backstories still need to be not obviously guided by the simple need to murder everyone and have non-murderbone paths for roleplay.